yarn.lock

```diff
@@ -8566,13 +8566,15 @@ performance-now@^2.1.0:
   version "2.1.0"
   resolved "https://registry.yarnpkg.com/performance-now/-/performance-now-2.1.0.tgz#6309f4e0e5fa913ec1c69307ae364b4b377c9e7b"
 
-photon@2.0.1:
-  version "2.0.1"
-  resolved "https://registry.yarnpkg.com/photon/-/photon-2.0.1.tgz#4044c1ad637ae4733744bb4643fb7b15a4429f97"
+photon@2.1.0:
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/photon/-/photon-2.1.0.tgz#a8d9aecdc91f8b9b267f3782a15892d5e9c4eb20"
+  integrity sha512-ouotmk51C1Ve4K0w3/t24qePIYbB8wMIc3fgzOyo5RaLUygcaQJQUjf2YimmN5AXl8UGnxP13cO2I3EYMjgR/w==
   dependencies:
     crc32 "0.2.2"
-    debug "3.1.0"
```

Comment:
I expected to see `debug@3.1.0` removed from the yarn.lock, but I don't know exactly how yarn.lock works.